docs/user_manual/calculations.md

```diff
@@ -419,31 +445,69 @@ for the next iteration:
   the measured voltage magnitude.
 - In iteration $k$, follow the steps below:
   - Linearize the voltage measurements by using the phase angle of the calculated voltages of iteration $k-1$.
   - Linearize the complex power flow measurements by using either the linearized voltage measurement if the bus is measured, or
   the voltage phasor result from iteration $k-1$.
-  - Compute the temporary new voltage phasor $\underline{\tilde{U}}^{(k)}$ using the pre-factorized matrix.
-  [Matrix-prefactorization](./performance-guide.md#matrix-prefactorization)
+  - Compute the temporary new voltage phasor $\underline{\tilde{U}}^{(k)}$ using the pre-factorized matrix. See also [Matrix-prefactorization](./performance-guide.md#matrix-prefactorization)
   - Normalize the voltage phasor angle by setting the angle of the slack bus to zero:
   - If the maximum deviation between $\underline{U}^{(k)}$ and $\underline{U}^{(k-1)}$ is smaller than the error tolerance $\epsilon$,
   stop the iteration. Otherwise, continue until the maximum number of iterations is reached. 
   
 In the iteration process, the phase angle of voltages at each bus is updated using the last iteration;
 the system error of the phase shift converges to zero. Because the matrix is pre-built and
-pre-factorized, the computation cost of each iteration is much smaller than Newton-Raphson
+pre-factorized, the computation cost of each iteration is much smaller than for the [Newton-Raphson](#newton-raphson-state-estimation)
 method, where the Jacobian matrix needs to be constructed and factorized each time.
 
 ```{warning}
-The algorithm will assume angles to be zero by default. This produces more correct outputs when the system is observable, but will
-prevent the calculation from raising an exception if it is unobservable, therefore giving faulty results.
+The algorithm will assume angles to be zero by default (see the details about voltage sensors). This produces more correct outputs when the system is observable, but will prevent the calculation from raising an exception, even if it is unobservable, therefore giving faulty results.
 ```
 
-Algorithm call: `CalculationMethod.iterative_linear`
+#### Newton-Raphson state estimation
 
-### Short circuit algorithms
+```warning
+This feature is still experimental and is at the time of writing not yet publicly available.
+```
+
+Algorithm call: {py:class}`CalculationMethod.newton_raphson <power_grid_model.enum.CalculationMethod.newton_raphson>`
+
+The Newton-Raphson approach solves state estimation by considering it as a system of real, non-linear equations.
+It then iteratively solves the first order Taylor expansion of that system. It does not make any linearization assumptions.
+It may therefore be more accurate and is likely to achieve convergence in less iterations than the [iterative linear](#iterative-linear-state-estimation) approach.
```

Comment:
```suggestion
It could be more accurate and converge in less iterations than the [iterative linear](#iterative-linear-state-estimation) approach.
```

Reply:
The reason that it could be more accurate is unclear from context, hence removing `Therfore`.